docs/release-notes/relnotes-2103.md

```diff
@@ -0,0 +1,64 @@
+# Release Notes
+
+## Devops Docker Builds, Version 2103 (March 2021)
+
+### New Features
+
+- **Images run as non-privileged user by default**
+
+      _Critical:_ We've greatly improved the security of our images by having them run as a non-privileged user by default.  See [Migrating to Unprivileged Images](https://devops.pingidentity.com/how-to/migratingRootToUnprivileged/) for information about migrating existing deployments.
+
+- **Layer simplification**
+
+      We've consolidated layers in our images where possible, greatly reducing image sizes.
```

Comment:
Change to 'We've consolidated layers in our images where possible' 
This change didn't reduce image size